template.yml

```diff
@@ -96,6 +96,127 @@ Resources:
           /category/delete/{category_id}:
             delete: *standardImplementation
 
+  # The AWS S3 bucket into which our SPA build files will be deployed
+  S3Bucket:
```

Comment:
Can we name this something more specific?

Reply:
Yes! How does UI-React-App sound?